index.js

```diff
@@ -57,56 +58,102 @@ module.exports = {
     "block-spacing": "error",
     "brace-style": ["error", "1tbs", { "allowSingleLine": true }],
     "camelcase": "off",
+    "comma-dangle": ["error", "always-multiline"],
```

Comment:
I personally am not a huge fan of this. I tend to like the JSON schema look/feel for JSON dictionaries.

Reply:
Trying out this rule in okta-ui-react, and realizing it's resulting in asking for changing certain test React code, for example:
```tsx
const testComponent = mount(
  <Component>
    Test
  </Component>
);
```
It would ask for a trailing comma after `</Component>`... I think the trailing comma looks a little funny with jsx 😆 

I'm thinking about changing this to be `always-multiline` for everything except for `functions`: `["error", { "arrays": "always-multiline", "objects": "always-multiline", "imports": "always-multiline", "exports": "always-multiline" }]`

Reply:
:joy: good changes here IMO. Especially since we have some apps that aren't being built with ES2017 yet I believe.